playwright/package.json

```
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build",
+    "test:e2e": "playwright test"
+  },
+  "dependencies": {
+    "@remix-run/node": "*",
+    "@remix-run/react": "*",
+    "@remix-run/serve": "*",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Let's keep this the same as in `_template`

```suggestion
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
```

---

sure! its done now.